packages/ingest/src/S3ProviderClient.ts

```diff
@@ -61,22 +61,28 @@ class S3ProviderClient implements ProviderClient {
   /**
```

Comment:
Updates to this method are the 'guts' of the change, we're adding a bucket parameter to this expected params for this sync method to allow for an override to the Provider bucket value (defined by the provider configuration) on a per-file basis